src/app/settings/settings.component.ts

```diff
@@ -46,4 +47,34 @@ export class SettingsComponent implements OnInit {
     debug('setting new server to ' + value);
     this.data.changeFhirMicroService(value);
   }
+
+  async createQuestionnaireResponse() {
+    const existingQuestionnaireResponseBundle = await this.data
+      .getFhirClient()
+      .resourceSearch({
+        resourceType: 'QuestionnaireResponse',
+        searchParams: {
+          questionnaire:
+            'http://fhir.ch/ig/ch-rad-order/Questionnaire/QuestionnaireRadiologyOrder',
+          identifier: 'http://ahdis.ch/fhir/Questionnaire|DEFAULT',
+        },
+      });
+
+    const id = existingQuestionnaireResponseBundle?.entry?.[0]?.resource?.id;
+    if (id) {
+      await this.data.getFhirClient().update({
+        id,
+        resourceType: 'QuestionnaireResponse',
+        body: {
+          ...questionnaireRadiologyOrderDefaultResponse,
```

Comment:
is this radiology specific code just here for testing?

Reply:
Yes, this is just for the proof-of-concept to be easily able to update the default response.